<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bảng chọn tài khoản để upload</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/logo/icon-sw.png</normaloff>:/logo/icon-sw.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutForTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="doIt">
           <property name="text">
            <string>Thực hiện</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tickAllBtn">
           <property name="text">
            <string>Chọn tất cả</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="untickAllBtn">
           <property name="text">
            <string>Bỏ chọn tất cả</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearAllUploader">
           <property name="text">
            <string>Xoá toàn bộ trong bảng chính</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">margin-top:10px</string>
      </property>
      <property name="text">
       <string>Bạn đang chuẩn bị áp dụng cho các video có thứ tự trong bảng dưới đây:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="whichIndex">
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-left:1px solid gray;
border-right:1px solid gray;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
